src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -428,6 +414,84 @@ describe('OnlineGameWrapperComponent of Quarto:', () => {
         expect(wrapper.currentPart).toEqual(CURRENT_PART);
         tick(wrapper.joiner.maximalMoveDuration * 1000);
     }));
+    describe('Move victory', () => {
+        it('Victory move from player should notifyVictory', fakeAsync(async() => {
+            //  Given a board on which user can win this move
+            const move0: QuartoMove = new QuartoMove(0, 3, QuartoPiece.AAAB);
+            const move1: QuartoMove = new QuartoMove(1, 3, QuartoPiece.AABA);
+            const move2: QuartoMove = new QuartoMove(2, 3, QuartoPiece.BBBB);
+            const move3: QuartoMove = new QuartoMove(0, 0, QuartoPiece.AABB);
+            await prepareBoard([move0, move1, move2, move3]);
+            componentTestUtils.expectElementNotToExist('#winnerIndicator');
+
+            // when doing wimming move
+            spyOn(partDAO, 'update').and.callThrough();
+            const winningMove: QuartoMove = new QuartoMove(3, 3, QuartoPiece.ABAA);
+            await doMove(winningMove, true);
+
+            // then we game should be a victory
```

Comment:
we -> the

Reply:
mais non t'as pas compris la phrase hein! (fait)